### PR TITLE
#167891584 Notifications for new travel request(email notification)

### PR DESCRIPTION
**Pull request by Pomile**
#### What does this PR do?
Have an email notification sent to admin users when new travel request is initiated

#### Description of Task to be completed?
- [x] Create Notifications model and migration
- [x] Create Reads model and migration
- [x] Create an HTML template for travel request email notification
- [x] Create a function that finds all admins
- [x] Build template with data
- [x] Send email to administrators 
- [x] Add notification data to the database

##### Unit test
1. Clone the project repository
2. Cd into the project repository
3. Run `git pull`
4. Run `git checkout ft-emailNotification-forNewTravelRequest-167891584`
5. Run `npm install`
6. Create database `database_name`
7. Create .env file
8. Run npm test

##### Unit test
1. Clone the project repository
2. Cd into the project repository
3. Run `git pull`
4. Run `git checkout  ft-emailNotification-configOpt-endpoint-167891584`
5. Run `npm install`
6. Create database `database_name`
7. Create .env file
8. Run npm test

##### Development test with Postman
 1. Run `npm run devstart`
 2. Launch postman
 3. Create a new post request with  `/api/v1/trips` 
 4. Add a request body `{
    "type": "oneway",
    "from": "xxxxxxxxxxxxxxxxx",
    "departureDate": "2000-09-09",
    "reason": "Just wanna travel",
    "destination":
    {
      "to": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
      "accomodation": "xxxxxxxxxxxxxxxxxxxxxxxxxxxx"
    }
  }`
#### What are the relevant pivotal tracker stories?
[#167891584](https://www.pivotaltracker.com/story/show/167891584)
#### Any background context you want to add?
Nil
#### Screenshots
Nil

### CODE REVIEW

**Review by max-wel — COMMENTED** [review]
I noticed that the ```Reads``` table contains read notifications. What if you added a field in the notifications table to handle that instead

File: src/database/migrations/20190905031234-create-notifications.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Notifications', {
+      id: {
```
Comment (max-wel) [inline src/database/migrations/20190905031234-create-notifications.js]: Kindly add a field that contains the notification recipient like ```recipientId```

Comment (Pomile) [reply src/database/migrations/20190905031234-create-notifications.js]: Thank you, Maxwell. 
Who should be the recipient? I understand that at the present we do not have a department manager. Why do you think need we `recipientId` in the Notification table? What if I want to send a notification to a group of users?

Comment (max-wel) [reply src/database/migrations/20190905031234-create-notifications.js]: The user the notification is meant for. I think it's needed so we can fetch all notifications belonging to a particular user

**Review by daniellamarr — APPROVED** [review]

**Pull request merged**